m2m-transfers/V3/m2m-transfers-json-contract-v3.md

```diff
@@ -376,18 +376,42 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
 
 ### Error Codes
 
-| errorCode | loggingErrorMessage                       |
-| --------- | ----------------------------------------- |
-| 501       | Config file read / validation error       |
-| 502       | Invalid source account                    |
-| 503       | Invalid recipient account                 |
-| 504       | Insufficient information                  |
-| 505       | Invalid input                             |
-| 506       | Error creating/deleting recipient         |
-| 507       | Error creating/updating/deleting transfer |
-| 508       | Undefined error                           |
-| 509       | Member unverified                         |
-| 510       | AccountId incorrect                       |
+*See Logging Error Message section for additional details
+
+| Error Code | Logging Error Message                                                       | Notes |
+| -----------|------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode ||
+| 501        | Config file error: [configuration file name] open error - [system generated letter file read error message] | This error represents issues with the configuration letter file setup |
+|            | Config file error: [configuration file name] read error - [system generated letter file read error message] ||
+|            | Config file error: 3-Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
+|            | Config file error: Invalid Parameter in CFG file ||
+| 502        | Invalid Source Account: Pref Access type 3 not found ||
+|            | Invalid Source Account: Acct Warning 1234 ||
+|            | Invalid Source Account: No eligible transfer from shares/loans ||
+|            | Invalid Source Account: Acct Type 1234 ||
```

Comment:
Should each of these error codes have some sort of additional notes? I know that many of the logging error messages are quite descriptive, but it makes it seem like some of the info is missing.  OR... maybe change the table header for that column from 'notes' to 'additional notes as needed'  ???  Other tables as well.

Reply:
changed column from 'notes' to 'additional notes as needed'